README.md

```diff
@@ -33,14 +33,24 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+    
+    Semantic HTML provides depth/meaning to webpages (each aspect is defined) and later styled accordingly with CSS.
 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+    
+    Block always starts on a new line and takes up all space. Inline goes side-by-side and only takes up the space needed.
 
 3. What are the 4 areas of the box model?
+    
+    height, padding, border, margin
 
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+    
+    main axis
```

Comment:
The align item center is on the cross axis :  
         y-axis  
          |
<---[item]---x-axis-->   align items moves it vertically(up and down) where the x and y meet.